### ISSUE TITLE
Future handling of not supported HTML tags

### ISSUE THREAD

**Issue by AileenCGN** [jbhannah/amperize#63]
Hi @jbhannah,

at the moment, we're only transforming a few HTML tags into AMP HTML. I would love to add more transformations (e. g. audio and video) in the near future. Nevertheless, some tags will still remain prohibited by AMP.

Here's the full list of restricted HTML tags by AMP: https://github.com/ampproject/amphtml/blob/master/spec/amp-html-format.md#html-tags

I would like to know, if you are planning to strip out prohibited HTML tags, so the output will be a 100% valid AMP HTML? Maybe with the help of the [Sanitize-HTML](https://github.com/punkave/sanitize-html) module?

Would love to hear your opinion about that!

Cheers, Aileen

**Comment by jbhannah** [comment]
Yes, ideally that will be one thing that this package does. For v1.0.0, its output should be fully valid AMP HTML.

**Comment by kirrg001** [comment]
[amp twitter](https://github.com/ampproject/amphtml/blob/master/extensions/amp-twitter/amp-twitter.md) might be interesting as well.

**Comment by wtubog** [comment]
Any updates on this?

**Comment by Luen** [comment]
I would really like to see support for the <video> tag to be added. Currently using ghost and it seems like it's getting ripped out because of this. Thanks
